admin/tools/docker/deployment/kubernetes/kubectl/admin/example/run-case01-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Launch Qserv multinode tests on Swarm cluster
```

Comment:
not using Swarm anymore -- maybe update comment to avoid confusion?

Reply:
:+1: